src/airdrop.js

```diff
@@ -1,13 +1,15 @@
 import tokens from '@aragon/templates-tokens'
```

Comment:
Let's make sure we publish this too...

Reply:
Already published v1.1.1, this could also be used for loading the token list in the finance app